aura/src/Aura/Languages.hs

```diff
@@ -39,31 +39,19 @@ along with Aura.  If not, see <http://www.gnu.org/licenses/>.
 
 -}
 
-module Aura.Languages where
+module Aura.Languages
+    ( module Aura.Languages
```

Comment:
[Indeed!](https://stackoverflow.com/questions/17241491/haskell-hide-specific-functions-in-module-exports)

I see the issue now. Your currently formulation is fairly clean then, given the circumstances.